### ISSUE TITLE
Potential issue? calling a kick on a name on an stolen/impossible name

### ISSUE THREAD

**Issue by surepy** [PazerOP/tf2_bot_detector#40]
Can't really check right now. but the message 
`Attention! There is a cheater on the other team named %s. Please kick them!`
can pretty sure cause potential issues if the name is stolen, involves newline, etc. might want to change it to

`Attention! There is a cheater on the other team named %s [id: (steamid)]. Please kick them!`
or craft another chat message with steamid only like
`[TF2BD] Votekick id: steamid` for example.

**Comment by KingTerrytheTerrible** [comment]
Ideally, you'd want to use the userid as that's what you use to votekick in console.

**Comment by surepy** [comment]
> Ideally, you'd want to use the userid as that's what you use to votekick in console.

yes, and using names on chat to communicate between TF2BD clients may cause potential issues and is the reason why I opened this issue.

**Comment by ClusterConsultant** [comment]
Related to #70

**Comment by ClusterConsultant** [comment]
If/when #70 comes about this will probably be closed since there wont be a ton of purpose to uber robust built in messages.
